build-ps/rpm/mysql-5.7-sharedlib-rename.patch

```diff
@@ -35,6 +35,7 @@ diff -rup old/client/CMakeLists.txt new/client/CMakeLists.txt
  SET_SOURCE_FILES_PROPERTIES(mysqltest.cc PROPERTIES COMPILE_FLAGS "-DTHREADS")
 -TARGET_LINK_LIBRARIES(mysqltest perconaserverclient regex)
 +TARGET_LINK_LIBRARIES(mysqltest mysqlclient regex)
```

Comment:
@EvgeniyPatlan @abychko we are changing perconaserverclient to mysqlclient here - what is the background to this change?